docs/guides/universal-receiver-delegate/set-default-implementation.md

```diff
@@ -184,12 +186,13 @@ const universalProfileURD = await universalProfileURDFactory
 ```typescript title="Deploy a Universal Receiver Delegate for the Universal Profile"
 const deployUniversalProfileURD = async () => {
   // create an instance of the LSP1UniversalReceiverDelegateUP
-  let universalProfileURD = new web3.eth.Contract(
+  const universalProfileURD = new web3.eth.Contract(
     LSP1UniversalReceiverDelegateUP.abi,
   );
+  let universalProfileURDAddress;
 
   // deploy the Universal Receiver Delegate UP contract
-  const universalProfileURDAddress = await universalProfileURD
```

Comment:
why deleting this and putting the variable above? Why not assigning the value on declaration like before? `const` is also safer as it prevents the value from being changed.

Reply:
because the value doesn't get returned properly, return type is void.